docs/serverless-v2/08-03-bind-service-instance.md

```diff
@@ -145,6 +151,57 @@ Follows these steps:
 
     > **NOTE:** If you added the **REDIS_** prefix for environmental variables in step 6, all variables will start with it. For example, the **PORT** variable will take the form of **REDIS_PORT**.
 
+    </details>
+    <details>
+    <summary label="console-ui">
+    Console UI
+    </summary>
+
+> **NOTE:** Serverless v2 is an experimental feature, and it is not enabled by default in the Console UI. To use its **Functions [preview]** view, enable **Experimental functionalities** in the **General Settings** view before you follow the steps.
+
+To create a binding, you must first create a sample service instance to which you can bind the lambda. Follow the sections and steps to complete this tutorial.
+
+### Provision a Redis service using an Addon
+
+> **NOTE:** If you already have a Redis instance provisioned on your cluster, move directly to the **Bind the lambda with the service Instance** section.
+
+Follow these steps:
+
+1. Select a Namespace from the drop-down list in the top navigation panel where you want to provision the Redis service.
+2. Go to the **Addons** view in the left navigation panel and select **Add New Configuration**.
+3. Enter `https://github.com/kyma-project/addons/releases/download/0.11.0/index-testing.yaml` in the **Urls** field. The Addon name is automatically generated.
+4. Select **Add** to confirm changes.
+
+    You will see that the Addon has the `Ready` status.
+
+### Create a Service Instance
+
+1. Go to the Catalog view where you can see the list of all available Addons and select **Redis**.
+2. Select **Add** to provision the Redis ServiceClass and create its instance in your Namespace.
+3. Change the **Name** to match the lambda, select `micro` from the **Plan** drop-down list, and set **Image pull policy** to `Always`.
+
+    > **NOTE:** The Service Instance, Service Binding, and Service Binding Usage can have different names than lambda, but it is recommended that all related resources share a common name.
+
+4. Select **Create** to confirm changes.
+
+    Wait until the status of the instance changes from `PROVISIONING` to `RUNNING`.
+
+### Bind the lambda with the service Instance
+
+1. Go to the **Functions [preview]** view at the bottom of the left navigation panel and select the lambda you want to bind to the Service Instance.
+2. Select **Select Service Bindings** in the **Service Bindings** section.
+3. Select the Redis service from the **Service Instance** drop-down list, add `REDIS_` as **Prefix for injected variables**, and make sure **Create new Secret** is checked.
```

Comment:
```suggestion
3. Select the Redis service from the **Service Instance** drop-down list, add `REDIS_` as **Prefix for injected variables**, and make sure you checked **Create new Secret**.
```

Reply:
It's selected by default, so in theory, you shouldn't do anything on your own as it should be done already.